includes/methods/Cgi.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <Request.hpp>
+#include <Host.hpp>
+#include <methods/AMethod.hpp>
+
+namespace Webserver
+{
+	#define SYSTEM_CALL_ERROR -1
+	#define CHILD_PROCESS  0
+
+	class Cgi : public AMethod
+	{
+		public:
+			enum FDs
```

Comment:
There are builtin defines for this

Reply:
Yes but only for stdIn and std out not for pipe read and write ends, even though they represent the same valeus